.github/workflows/ci.yaml

```diff
@@ -234,8 +273,16 @@ jobs:
                     cat ${{ github.workspace }}/.github/workflows/ci.bazel5.bazelrc >> ${{ github.workspace }}/.github/workflows/ci.bazelrc
                   fi
 
-            - name: Write EngFlow credentials
-              # Writes EngFlow credential files for RBE configurations
+            - name: Configure Bazel version (Windows)
+              if: matrix.os == 'windows-latest'
+              working-directory: ${{ matrix.folder }}
+              # Overwrite the .bazelversion instead of using USE_BAZEL_VERSION so that Bazelisk
+              # still bootstraps Aspect CLI from configuration in .bazeliskrc. Aspect CLI will
+              # then use .bazelversion to determine which Bazel version to use.
+              run: echo "${{ matrix.bazelversion }}" > .bazelversion
```

Comment:
Why do we have a windows specific "Configure Bazel version"? It seems like we only drop the bazel5 setup in the windows one?